src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -562,3 +562,17 @@ export const noFDCWarning = (
   </div>
 );
 
+
+const options = [
+  { id: 1, label: 'first' },
+  { id: 2, label: 'second' },
+  { id: 3, label: 'third' },
+  { id: 4, label: 'fourth' },
+];
+
+
+// Mimic querying the api for options
+export function queryForFacilities(input = '', setOptions) {
+  // Emulate a fast api call
+  return Promise.resolve(setOptions(options.filter(o => o.label.includes(input))));
```

Comment:
I'll be writing another ticket to hook this up to the api. I was trying to keep this PR smaller. :man_shrugging: